01-messaging.md

```diff
@@ -119,24 +119,28 @@ For simplicity of diagnosis, it is often useful to tell the peer that something
 
 1. type: 17 (`error`)
 2. data:
-   * [8:channel-id]
+   * [32:funding-txid]
+   * [1:funding-output-index]
    * [2:len]
    * [len:data]
 
 The 2-byte `len` field indicates the number of bytes in the immediately following field.
 
 #### Requirements
 
+The channel is referred to by `funding-txid` and `funding-output-index`,
+unless `funding-txid` is 0, in which case `funding-output-index` of 0 refers
```

Comment:
Nice addition. As a minor nit, I think we should specify we mean "all zeroes" as in `'0'*32` for the `funding-txid` field to have a connection-wide scope.